src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -89,6 +89,8 @@ internal sealed class TextBufferIssueTracker : IIssueConsumer
             var projectName = this.ProjectItem?.ContainingProject.Name ?? "{none}";
             this.Factory = new SnapshotFactory(new IssuesSnapshot(projectName, this.FilePath, 0,
                 new List<IssueMarker>()));
+
+            document.FileActionOccurred += FileActionOccurred;
```

Comment:
Minor product change to fix a bug around file renaming - we need to start listening to file change events immediately to pick up file renames...